Readme.md

```diff
@@ -1,76 +1,71 @@
+Run tests:
 
-The more methods you put in your package, the better.
+`npm run test`
 
-*Deployment*:
+Coverage with NYC:
 
-Videos and screenshots will not be accepted. You must leave links to:
+`npm run coverage`
 
-1. Your NPM package
-2. A deployed project on Runkit or Codepen.
+_Note:_ you will see numbers that are not all 100% - this is on purpose. You can comment out a dummy `if(false)` lines, then run `npm test coverage` again and ensure the coverage is at 💯 percent.
 
+## Available Functions
 
-Your deployment options for #2 above are:
+### countElements()
 
-1. Runkit - You will have to sign up and create a new notebook. See the example [Runkit Notebooks](https://runkit.com/monarchwadia). 
-2. Codepen - You can import your project into Codepen using [unpkg](https://unpkg.com/)
+Given an array of integers, count the number of elements X such that (X + 1) is also in the array.
 
-## Bonus points
+Example:
 
-Bonus points will be awarded for:
-* Creating a typescript-compatible project
-* Creating unit tests
-* Using a precompiler (only if appropriate, i.e. if you are using Typescript or Babel, or if you are creating a React library)
-* Leaving detailed instructions for local development on `Readme.md`
-* Writing documentation (for your future self's benefit)
+Input: `[1,2,3]`
 
-## Frameworks:
+Output: 2
 
-Any or none. You can create a personal library with just plain JavaScript. You can also create a personal library that contains common React/Vue/Angular components that you find yourself using often across projects.
+Explanation: 1 and 2 are counted because (1 + 1) = 2 and (2 + 1 ) = 3 are in the array.
 
-## Restrictions
+### groupAnagrams()
 
-* Your project must use JavaScript.
-* You must follow deployment instructions above.
+Given an array of strings, group anagrams together.
 
-There are no other restrictions.
+Example:
 
-## Tips for success:
+Input: `["eat", "tea", "tan", "ate", "nat", "bat"]`
 
-1. The project is intentionally simple so you can focus on code quality.
-1. The requirements are intentionally minimalistic so you can get as creative as you'd like.
-1. You get a LOT of time for code review comments. So you can win even if you feel your project isn't great, simply on the strength of your code review comments!
-1. Keep in mind, 33% of your score is based on aesthetic appeal!
+Output:
 
-## Resources
-
-* [Creating and publishing unscoped public packages](https://docs.npmjs.com/creating-and-publishing-unscoped-public-packages)
-* [Video - Creating & publishing your own NPM modules](https://www.youtube.com/watch?v=rTsz09zRuTU)
-* [freeCodeCamp's Tutorial](https://www.freecodecamp.org/news/how-to-make-a-beautiful-tiny-npm-package-and-publish-it-2881d4307f78/)
+```
+[
+  ["ate","eat","tea"],
+  ["nat","tan"],
+  ["bat"]
```

Comment:
I think this README is well written and documented. There are detailed instructions on what is included in the library, what each do, examples, and testing coverage. My only suggestion would be to dress this README up with some badges that demonstrate what you're trying to get across here. I found some badges on https://shields.io/ that do this - test coverage, npm package version, languages used.